test/helpers/replicate.js

```diff
@@ -45,21 +85,8 @@ function setupFeedRequests(person1, person2) {
     ]),
```

Comment:
"This is the list of feedIds that we should care about in our vector clock" mix says